src/System.Management.Automation/engine/CoreAdapter.cs

```diff
@@ -3122,7 +3122,7 @@ internal static bool DisallowPrivateReflection(TypeInfo typeInfo)
             }
 
             var productAttribute = assembly.GetCustomAttribute<AssemblyProductAttribute>();
-            if (productAttribute != null && string.Equals(productAttribute.Product, "Microsoft� .NET Framework", StringComparison.OrdinalIgnoreCase))
+            if (productAttribute != null && string.Equals(productAttribute.Product, "Microsoft® .NET Framework", StringComparison.OrdinalIgnoreCase))
```

Comment:
Can we change this to `Microsoft\u00a9` (assuming that is the correct character). When we run automated tools, they always mess this character up.

Reply:
We have the character in some files - maybe fix this all in follow PR?

Reply:
Created issue https://github.com/PowerShell/PowerShell/issues/4752 as we should just fix that (and potentially Trademark as well)